src/main.ts

```diff
@@ -245,19 +221,21 @@ async function main() {
 	//
 	// Substrate Reference:
 	// - Staking Pallet: https://crates.parity.io/pallet_staking/index.html
+	// - `RewardDestination`: https://crates.parity.io/pallet_staking/enum.RewardDestination.html
+	// - `Bonded`: https://crates.parity.io/pallet_staking/struct.Bonded.html
 	// - `StakingLedger`: https://crates.parity.io/pallet_staking/struct.StakingLedger.html
-	get('/staking/:address', async (params) => {
+	get('/accounts/:address/staking', async (params) => {
```

Comment:
I'm not really comfortable with this. Even if it's the direction we want to go, now this pattern is broken across all of sidecar. This should be a separate discussion around the sidecar API where we change everything.